<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #121212;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: transparent;
    color: white;
    font-size: 32px;
    border: none;
    text-align: right;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_history">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>341</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: #121212;
    color: white;
    gridline-color: transparent;
    border: none;
    font-size: 20px;
}

QTableWidget::item {
    font-size: 24px;
    font-weight: bold;
    color: #FFA726;
}
</string>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>431</width>
      <height>336</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_7">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_8">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_9">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_borrar_ultimo_numero">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_4">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_5">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_6">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_resta">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_1">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_2">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_3">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_suma">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_coma">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_0">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_dividir">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_multiplicacion">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btn_clear">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="btn_igual">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1E1E1E;
    color: white;
    border: none;
    font-size: 24px;
    padding: 15px;
}

QPushButton:hover {
    background-color: #333333;
}

/* Botón de igual (=) */
QPushButton#btn_igual {
    background-color: #FF9800;
    color: black;
    font-weight: bold;
}

/* Botón de eliminar (DEL) */
QPushButton#btn_del {
    background-color: #D32F2F;
    color: white;
    font-weight: bold;
}

/* Botón de limpiar (Clear) */
QPushButton#btn_clear {
    background-color: #FF5722;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
